spec/requests/api/user_can_search_for_movie_by_title_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::MoviesController, type: :request do
+  let!(:registered_user) { create(:user) }
+  let!(:authorization_headers) { registered_user.create_new_auth_token }
+
+  describe 'Search for a movie by title' do
+    context 'When users provide query params' do
+      before do
+        fixture_file =
+          File.open("#{::Rails.root}/spec/fixtures/movies_search.json").read
+        query = 'christmas'
+        stub_request(
+          :get,
+          "https://api.themoviedb.org/3/search/movie?api_key=#{
+            Rails.application.credentials.moviedb[:api_key]
+          }&language=en-US&query=#{query}&page=1&include_adult=false"
+        ).to_return(status: 200, body: fixture_file, headers: {})
```

Comment:
Why are you adding the fixture in here when you had the other fixture in your rails helper?

Reply:
This is because we need to search movies by titles and use the exact link from the movie database with query. The one in rails helper is for only to display the popular movies.